config/config.sample.php

```diff
@@ -278,11 +277,11 @@
   ],
 
 /**
- * Define a custom link to reset password
+ * Define a custom link to reset passwords
  * If your user backend does not allow password resets (e.g. when it's a read-only
  * user backend like LDAP), you can specify a custom link, where the user is
  * redirected to, when clicking the "reset password" link after a failed login-attempt.
- * In case you do not want to provide any link, replace the url with 'disabled'
+ * In case you do not want to provide any link, replace the URL with 'disabled'.
```

Comment:
```suggestion
 * If you do not want to provide any link, replace the URL with 'disabled'.
```
optional suggestion - IMO "If" is more natural English than "In case"